pkg/policies/opa/rego/docker/docker_expose/exposeport.rego

```diff
@@ -0,0 +1,7 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[run.id]{
+	run := input.expose[_]
+	config := run.config
+    contains(config, "22")
```

Comment:
indentation issue

value of `config` variable may or may not be in form of an array, need to create checks for both possibilities. please keep edge cases in mind while writing policy code

Reply:
okay, on it